style.css

```diff
@@ -541,6 +679,80 @@ nav a {
     margin-top: 28px;
   }
 
+  /************* pop-up window *****************/
+  .pop-up {
+    width: 716px;
+  }
```

Comment:
Great design team 🤝

- [ ] [Optional] Kindly ensure your design is a replica of the design given. I would suggest you add a margin to the bottom of your desktop popup so that the buttons don't appear to be overflowing from the design.
![desktop-popup](https://github.com/Fombi-Favour/Microverse-portfolio/assets/63917725/ebf9339a-f29f-47a2-a097-26665fe98ff7)